cli/build.rs

```diff
@@ -330,6 +330,21 @@ fn main() {
   if target != host {
     panic!("Cross compiling with snapshot is not supported.");
   }
+
+  #[cfg(target_os = "windows")]
+  println!(
+    "cargo:rustc-link-arg-bin=deno=/DEF:{}",
+    std::path::Path::new("exports.def")
+      .canonicalize()
+      .expect(
+        "Missing exports.def! Generate using tools/napi/generate_link_win.js"
+      )
```

Comment:
Can this step be automated somehow?

Reply:
yes we can run it in build.rs but it's rare that exports.def needs an update.